game/src/core/utils/components/draw/CoreAnimationPathEnum.java

```diff
@@ -0,0 +1,16 @@
+package core.utils.components.draw;
+
+/**
+ * This enum stores the default paths to the animations used by the systems inside the core package.
+ */
+public enum CoreAnimationPathEnum implements IAnimationPathEnum {
+    IDLE_LEFT("/idleLeft"),
+    IDLE_RIGHT("/idleRIgth"),
+    RUN_LEFT("/runLeft"),
+    RUN_RIGHT("/runRight");
```

Comment:
Ich würde das ohne den "/" am Anfang schreiben und diesen dann mit `File.pathSeparator` an der passenden Stelle zusammenbauen. Dann klappt das auf jedem Betriebssystem :)

Außerdem machst Du in `DrawComponent` sowas: `animationMap.get(CoreAnimationPathEnum.IDLE_LEFT)`... Und in der Map ist der Key einfach nur der Name des Ordners und nicht "/Ordner".